chart/values.yaml

```diff
@@ -1314,9 +1314,96 @@ statsd:
 
   priorityClassName: ~
 
+  histogramBuckets:
+    - 0.005
+    - 0.01
+    - 0.025
+    - 0.05
+    - 0.1
+    - 0.25
+    - 0.5
+    - 1
+    - 2.5
+
+  defaultMappings:
+    # Map dot separated stats to labels
+    - match: airflow.dagrun.dependency-check.*.*
+      name: "airflow_dagrun_dependency_check"
+      labels:
+        dag_id: "$1"
+    - match: airflow.ti_failures
+      name: "airflow_ti_tasks_failures_total"
+    - match: airflow.ti_successes
+      name: "airflow_ti_tasks_succeeded_total"
+    - match: airflow.ti.start.*.*
+      name: "airflow_ti_tasks_started_total"
+      labels:
+        dag_id: "$1"
+        task_id: "$2"
+    - match: airflow.ti.finish.*.*.*
+      name: "airflow_ti_tasks_finished_total"
+      labels:
+        dag_id: "$1"
+        task_id: "$2"
+        task_state: "$3"
```

Comment:
These are all new, right? I think this will also break backwards compatibility, no?